docs/metadata-store/enabling-khepri.md

```diff
@@ -0,0 +1,212 @@
+user should be aware of.
+
+:::info
+Khepri is very new in RabbitMQ 4.0.x. Some behaviors change and it may be
+unstable and/or slow for your workload. We encourage you to test thoroughly
+before enabling Khepri in production.
+
+That said, **Khepri is fully supported**. It will be **possible to upgrade
+from 4.0.x to future releases** with Khepri enabled.
+:::
+
+:::note
+The Feature flags subsystem uses the words *stable* and *experimental* to
+qualify feature flags maturity.
+
+An *experimental* feature flag is used in two situations:
+1. To introduce changes to get feedback early during the development. These
+   changes could be reverted, upgrading a RabbitMQ node with such a feature
+   flag enabled may not bo possible and support may not be provided.
+2. For features the RabbitMQ team committed to and provides support for, until
+   it is ready to be enabled by default, possibly replacing an older system.
+
+Khepri in RabbitMQ 3.13.x was in the first group. Be reassured that Khepri in
+RabbitMQ 4.0.0 and onward is in that second group and is therefore fully
+supported.
+:::
+
+## On a brand new RabbitMQ node
+
+### Using the CLI
+
+1.  Start the new RabbitMQ node using a method of your choice. The example
+    below executes the [`rabbitmq-server(8)` command](../man/rabbitmq-server.8)
+    directly:
+
+    ```
+    rabbitmq-server
+    ```
+
+    At that point, the **node is using Mnesia** as the metadata store backend.
+
+2.  Enable the `khepri_db` feature flag:
+
+    ```
+    rabbitmqctl enable_feature_flag --experimental khepri_db
+    ```
+
+See the next page to learn more about what happens when nodes with Mnesia and
+nodes with Khepri are clustered together.
+
+### Using the management UI
+
+1.  Start the new RabbitMQ node using a method of your choice. See [the
+    example above](#using-the-cli).
+
+    At that point, the **node is using Mnesia** as the metadata store backend.
+
+2.  Enable the [management plugin](../management):
+
+    ```
+    rabbitmq-plugins enable rabbitmq_management
+    ```
+
+3. Open and log into the [management UI](../management#usage-ui).
+
+4. Navigate to *"Admin > Feature Flags"*.
+
+5. Tick *"I understand the risk"* and click the *"Enable"* button:
+
+    <figure className={diagramStyles.diagram}>
+    <EnableInUI/>
+    <figcaption>The experimental feature flags section in the management
+    UI</figcaption>
+    </figure>
+
+### Using an environment variable
+
+You can set the `$RABBITMQ_FEATURE_FLAGS` environment varable to set the list
+of feature flags to enable at boot time on a new node. The variable must be
+set to the exhaustive list of feature flags to enable on this node. This
+variable is considered on the very first boot only; it is ignored afterwards.
+
+:::warning
+The use of this variable requires some caution: because the variable takes an
+exhaustive list, you must be careful to leave a feature flag disabled that
```

Comment:
Should this be "not to leave a feature flag disabled"?

Reply:
You are right, fixed.